.github/workflows/ci.yml

```diff
@@ -0,0 +1,17 @@
+name: CI
+on:
+  pull_request:
+  push:
+    branches:
+      - master
```

Comment:
```suggestion
```

Maybe remove this branch restriction? That way contributors can test their feature branches *before* creating potentially failing PRs.

Reply:
True. I think in this case the cleanest syntax becomes
```yaml
on: [push, pull_request]
```
I'll commit this